src/watchmaker/workers/salt.py

```diff
@@ -89,7 +89,8 @@ def __init__(self, *args, **kwargs):
         # Pop arguments used by SaltBase
         self.user_formulas = kwargs.pop('user_formulas', None) or {}
         self.computer_name = kwargs.pop('computer_name', None) or ''
-        self.ent_env = kwargs.pop('environment', None) or ''
+        self.ent_env = kwargs.pop('environment', None) or None
```

Comment:
What's the reasoning behind changing the default here from `''` to `None`?

Reply:
Fixed